src/components/__tests__/Result.test.js

```diff
@@ -0,0 +1,176 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Result from '../sections/Result';
+
+const initState = {
+  name: 'bulbasaur',
+  avatar:
+    'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/1.svg',
+  weight: 69,
+  height: 7,
+  types: [
+    {
+      slot: 1,
+      type: {
+        name: 'grass',
+        url: 'https://pokeapi.co/api/v2/type/12/',
+      },
+    },
+    {
+      slot: 2,
+      type: {
+        name: 'poison',
+        url: 'https://pokeapi.co/api/v2/type/4/',
+      },
+    },
+  ],
+  abilities: [
+    {
+      ability: {
+        name: 'overgrow',
+        url: 'https://pokeapi.co/api/v2/ability/65/',
+      },
+      is_hidden: false,
+      slot: 1,
+    },
+    {
+      ability: {
+        name: 'chlorophyll',
+        url: 'https://pokeapi.co/api/v2/ability/34/',
+      },
+      is_hidden: true,
+      slot: 3,
+    },
+  ],
+  stats: [
+    {
+      base_stat: 48,
+      effort: 0,
+      stat: {
+        name: 'hp',
+        url: 'https://pokeapi.co/api/v2/stat/1/',
+      },
+    },
+    {
+      base_stat: 49,
+      effort: 0,
+      stat: {
+        name: 'attack',
+        url: 'https://pokeapi.co/api/v2/stat/2/',
+      },
+    },
+    {
+      base_stat: 50,
+      effort: 0,
+      stat: {
+        name: 'defense',
+        url: 'https://pokeapi.co/api/v2/stat/3/',
+      },
+    },
+    {
+      base_stat: 78,
+      effort: 1,
+      stat: {
+        name: 'special-attack',
+        url: 'https://pokeapi.co/api/v2/stat/4/',
+      },
+    },
+    {
+      base_stat: 79,
+      effort: 0,
+      stat: {
+        name: 'special-defense',
+        url: 'https://pokeapi.co/api/v2/stat/5/',
+      },
+    },
+    {
+      base_stat: 80,
+      effort: 0,
+      stat: {
+        name: 'speed',
+        url: 'https://pokeapi.co/api/v2/stat/6/',
+      },
+    },
+  ],
+};
+
+describe('Test Result page', () => {
```

Comment:
regarding naming. does not make sense to use `test` or `check` since it's all about testing. I don't say there is the single correct naming approach but it's better to be consistent

IMO good enough approach which describes behaviour under test:

```
describe('renders', () => {
  it('name', () => ....)
  it('types', () => ....)
....)
```

or in more complex cases(but without nesting into `describe()`):

```
it('disables Send button after request is sent')
```

Definitely not a big deal, but at least build your naming schema and stick to it